governance/SIG Network Charter.md

```diff
@@ -1,97 +1,109 @@
-
-- Items that span or require other SIGs or groups and how it relates to this SIG’s responsibilities
+- Define and maintain network layer encryption, as a cross-cutting issue with `#sig-security`.
+- Define and maintain network testing, as a cross-cutting issue with `#sig-testing`.
+- Define and maintain cloud services testing for cloud service components accepted by SIG, as a cross-cutting issue with `#sig-testing`.
+- Items that span or require other SIGs or groups and how it relates to this SIG’s responsibilities.
 
 ## Out of Scope
-- Not responsible for cloud or network services or interopability with such.
-- Not responsible for 3rd party gems and code for network services
+- Not responsible for sample projects unless projects are approved/accepted by SIG.
+- Not responsible for cloud or network services or interoperability with such services.
+- Not responsible for 3rd party gems and code for network services. Contributors submitting such integrations are responsible for support.
 - Not responsible for best practices of network or services implementation.
-- Not responsible for the network tool implementation outside of core multiplayer and cloud services, but may advise teams on best practices.
+- Not responsible for networked tools implementation, outside of core responsibilities for multiplayer and cloud services, but may advise teams on best practices.
+- Versioning of communication protocols (not currently a responsibility of the SIG).
+- Support for peer-to-peer connectivity (not currently a responsibility of the SIG).
+- Definition and implementation of 3rd party distribution platform services interfaces.
 
-- Items that are optional or are not the responsibility of this SIG.
 
 ## SIG Links and lists:
-
-- Joining this SIG
-- Slack/Discord
-- Mailing list
-- Issues/PRs
-- Meeting agenda & Notes
+- Slack/Discord: See https://o3de.org/community/ for Discord link, join the conversation in `#sig-network`.
+- Mailing list: https://lists.o3de.org/g/sig-network
+- SIG repro: https://github.com/o3de/sig-network
+- Issues/PRs: 
+     - Issues: https://github.com/o3de/o3de/issues?q=is%3Aissue+is%3Aopen+label%3Asig%2Fnetwork
+     - PRs: https://github.com/o3de/o3de/pulls?q=is%3Apr+is%3Aopen+label%3Asig%2Fnetwork
+- Meeting agenda & Notes: 
+     - Agendas: https://github.com/o3de/sig-network/labels/mtg-agenda
+     - Meeting Notes: https://github.com/o3de/sig-network/tree/main/meetings 
 
 ## Roles and Organization Management
 
-SIG Docs adheres to the standards for roles and organization management as specified by <sig-governance>. This SIG opts in to updates and modifications to <sig-governance>
+SIG-network adheres to the standards for roles and organization management as specified by [O3DE Foundation](https://github.com/o3de/community). This SIG opts in to updates and modifications to This SIG opts in to updates and modifications to the Foundation's guidance (referred to below as <sig-governance>)
```

Comment:
duplicate? "This SIG opts in to updates and modifications to" is written 2x

Reply:
Opps, will fix. Thanks.